compiler/src/typed/typemod.re

```diff
@@ -578,43 +506,59 @@ let type_module = (~toplevel=false, funct_body, anchor, env, sstr /*scope*/) =>
       )
     ) {
     | Some(External_name(name)) =>
-      let export = {
-        pex_name: Location.mknoloc(Ident.name(List.hd(idents))),
-        pex_alias: Some(Location.mknoloc(name)),
-        pex_loc: Location.dummy_loc,
-      };
+      let export =
+        PProvideValue({
+          name:
+            Location.mknoloc(
+              Identifier.IdentName(
+                Location.mknoloc(Ident.name(List.hd(idents))),
+              ),
+            ),
+          alias:
+            Some(
+              Location.mknoloc(
+                Identifier.IdentName(Location.mknoloc(name)),
+              ),
+            ),
+          loc: Location.dummy_loc,
+        });
```

Comment:
This handles the `@externalName` attribute, and it doesn't have locations because attributes don't save location information. Could make an issue about that.

Reply:
Issue please 🙏

Reply:
https://github.com/grain-lang/grain/issues/1630